editor/src/components/canvas/canvas-strategies/strategies/keyboard-reorder-strategy.ts

```diff
@@ -101,15 +105,59 @@ export function keyboardReorderStrategy(
   }
 }
 
-function getIndexChangeDeltaFromKey(key: KeyCharacter, delta: number): number {
-  switch (key) {
-    case 'left':
-    case 'up':
-      return -delta
-    case 'right':
-    case 'down':
-      return delta
-    default:
-      return 0
-  }
+// This function creates the map which describes which keyboard cursor button should move the index which way.
+// In standard layouts keyboard up and left moves backward and keyboard down and right moves forward.
+// In flex reverse layouts this is fully reversed: keyboard up and left moves forward and keyboard down and right moves backward.
+// If you have rtl text direction on top of any layouts, that should switch the effect of the left and right keys (but leave up and down as it is)
+function getIndexChangesForArrowKeys(element: ElementInstanceMetadata | null): {
+  [key: string]: number
```

Comment:
nitpick but I think it's worth explicitly listing up right bottom left here

Reply:
I thought about this but then I have to cast the `KeyCharacter` to this union, and that just did not feel right.

Reply:
Fixed in acb802445b9968ef11286a5291bbb7340fa58b8a